pkg/operator/ceph/cluster/osd/labels.go

```diff
@@ -36,13 +36,19 @@ const (
 	OSDOverPVCLabelKey = "ceph.rook.io/pvc"
 	// TopologyLocationLabel is the crush location label added to OSD deployments
 	TopologyLocationLabel = "topology-location-%s"
+	// CephImageVersionLabel is the ceph image version label added to PVC
+	CephImageVersionLabelKey = "ceph.rook.io/CreatedCephImageVersion"
+	// RookVersionLabel is the rook image version label added to PVC
+	RookImageVersionLabelKey = "ceph.rook.io/CreatedRookVersion"
```

Comment:
```suggestion
	rookImageTagLabelKey = "ceph.rook.io/originalRookImageTag"
```

Reply:
I think `cluster.rookVersion` has Rook's image name including the tag. so how about `CreatedRookImage`? the reason I want to use `Created` is written in comments above.